### PR TITLE
KLIMA DAO UI Fixes, Settings menu, Routes full Information [PRO-1399, PRO-1408, PRO-1402]

### PR DESCRIPTION
**Pull request by parag-chauhan**
## Description
- PRO-1399: KLIMA DAO UI Fixes
  - Remove gray from unselectable sKLIMA (users cannot change it but it should be shown).
  - Fix how offer is presented:
  - Add LiFi logo to the route:
  - Action Preview
- PRO-1408: Settings menu
  - Tapping cog should close all open windows from settings
- PRO-1402: Display full route information when checking out routes

## How Has This Been Tested?
- Local testing, Compare with Design

### KLIMA DAO UI Fixes
image
image

### Display full route information when checking out routes
Peek 2023-05-25 13-08

### Tapping cog should close all open windows from settings
Peek 2023-05-25 13-09

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx [hunk 1]
```diff
@@ -446,14 +454,7 @@ const KlimaStakingTransactionBlock = ({
...
       />
-      <NetworkAssetSelectInput
-        label="To"
-        selectedNetwork={supportedChains[1]}
-        selectedAsset={klimaAsset}
-        disabled={true}
-        walletAddress={selectedAccountType === AccountTypes.Contract ? accountAddress : providerAddress}
-        accountType={selectedAccountType}
-      />
+      <NetworkAssetInfoCard label="To" networkDetails={supportedChains[1]} assetDetails={klimaAsset} />
```
Comment (MayukhSamadder2000) [inline src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx:457]: @parag-chauhan I don't think we need a separate component for this maybe you can pass a property to `<NetworkAssetSelectInput {...args} readOnly />` so this `readOnly` property should handle the things

**Review by MayukhSamadder2000 — CHANGES_REQUESTED** [review]
I dont think the newFileYou created NetworkAssetInfoCard is needed

File: src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx (hunk 1 above)
Comment (parag-chauhan) [reply src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx]: @MayukhSamadder2000 Thanks for the suggestion. Updated to code.

File: src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx [hunk 2]
```diff
@@ -31,15 +31,15 @@ import CombinedRoundedImages from '../Image/CombinedRoundedImages';
 import { DestinationWalletEnum } from '../../enums/wallet.enum';
 import { BulletList } from "react-content-loader";
 
-const Wrapper = styled.div<{ disabled: boolean, expanded?: boolean, hover?: boolean }>`
+const Wrapper = styled.div<{ disabled: boolean, expanded?: boolean, hover?: boolean, readOnly: boolean }>`
```
Comment (MayukhSamadder2000) [inline src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx:34]: Keep read-only as optional

Comment (parag-chauhan) [reply src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx]: @MayukhSamadder2000 Updated

**Review by MayukhSamadder2000 — APPROVED** [review]
Lgtm

**Pull request merged**